packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec

```diff
@@ -0,0 +1,89 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name pulp_rpm_client
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 3.0.0b7.dev01570381057
+Release: 1%{?dist}
+Summary: Pulp 3 RPM plugin API Ruby Gem
+Group: Development/Languages
+License: GPLv2
+URL: https://github.com/pulp/pulp_rpm/
```

Comment:
It is confusing to not find sources at this URL.

Reply:
there is no source location for the bindings, as they are generated from the api specification defined in the pulp_rpm repository